packages/react-avatar/MIGRATION.md

```diff
@@ -1,51 +1,54 @@
 
 In cases where migration is possible, the following props will need to be renamed:
 
 - `text` => `name`
 - `size` converted from an enum to a number, and not all previous sizes are supported:
-  - `PersonaSize.size8` => Not Supported. Use the `Badge` component since this size of Persona only displays the badge.
-  - `PersonaSize.size16` => Not Supported. Can use `size={20} tokens={{ width: '16px', height: '16px' }}`
+  - `PersonaSize.size8` => Not Supported. Use the `PresenceBadge` component since this size of Persona only displays the badge.
+  - `PersonaSize.size16` => Not Supported. Use `size={20}`, and optionally add `style={{ width: '16px', height: '16px' }}` if the exact size is needed.
   - `PersonaSize.size24` => `size={24}`
   - `PersonaSize.size28` => `size={28}`
   - `PersonaSize.size32` => `size={32}`
   - `PersonaSize.size40` => `size={40}`
   - `PersonaSize.size48` => `size={48}`
   - `PersonaSize.size56` => `size={56}`
   - `PersonaSize.size72` => `size={72}`
-  - `PersonaSize.size100` => Not Supported. Can use `size={96} tokens={{ width: '100px', height: '100px' }}`
+  - `PersonaSize.size100` => Not Supported. Use `size={96}`, and optionally add `style={{ width: '100px', height: '100px' }}` if the exact size is needed.
   - `PersonaSize.size120` => `size={120}`
 - `coinSize` => Use `size`
-- `imageUrl` => `image`
-- `imageAlt` => Set `alt` on the `image` slot
-- `imageInitials` => `label` (or remove and have it calculated from `name`)
-- `presence` => Use the `Badge` component
-- `presenceColors` => Use the `Badge` component
-- `presenceTitle` => Use the `Badge` component
-- `isOutOfOffice` => Use the `Badge` component
-- `showUnknownPersonaCoin` => NOT SUPPORTED - Set the `icon` prop to an appropriate icon
-- `initialsColor` => Use `color="colorful"` or specify a color by name like `color="darkRed"`
-- `showInitialsUntilImageLoad` => (This is the normal behavior of Avatar)
-- `imageShouldFadeIn` => NOT SUPPORTED
-- `imageShouldStartVisible` => NOT SUPPORTED
-- `onPhotoLoadingStateChange` => NOT SUPPORTED
-- `onRender*` => NOT SUPPORTED - add custom components to the slots instead of overriding the rendering
-  - (Or to render a square image, use the `shape='square` prop)
+- `imageUrl` => `image={{ src: '...' }}`
+- `imageAlt` => The Avatar's `aria-label` can be used.
+- `imageInitials` => `initials`
+- `presence` => `badge` (see the `PresenceBadge` component for more details)
+  - `PersonaPresence.none` => (Default)
+  - `PersonaPresence.away` => `badge={{ status: 'away' }}`
+  - `PersonaPresence.blocked` => Not Supported.
+  - `PersonaPresence.busy` => `badge={{ status: 'busy' }}`
+  - `PersonaPresence.dnd` => `badge={{ status: 'doNotDisturb' }}`
+  - `PersonaPresence.offline` => `badge={{ status: 'offline' }}`
+  - `PersonaPresence.online` => `badge={{ status: 'available' }}`
+- `presenceColors` => Not Supported. The badge could be restyled using CSS. E.g. `badge={{ style: { color: '...' } }}`
```

Comment:
Nit: Change "The badge could be restyled..." to "The badge can be restyled ...".